packages/fd-send-recv/fd-send-recv.2.0.2/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "xen-api@lists.xensource.com"
+authors: [
+  "David Scott"
+  "David Sheets"
+  "Euan Harris"
+  "Vincent Bernardoff"
+]
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/xapi-project/ocaml-fd-send-recv"
+bug-reports: "https://github.com/xapi-project/ocaml-fd-send-recv/issues"
+dev-repo: "git+https://github.com/xapi-project/ocaml-fd-send-recv.git"
+doc: "https://github.com/xapi-project/ocaml-fd-send-recv/blob/master/lib/fd_send_recv.mli"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+]
+depends: [
+  "ocaml" {>= "5.0"}
```

Comment:
Is it just ocaml5 or it supports also older versions? Unless you use effects it should at least work in 4.14.
```suggestion
  "ocaml" {>= "4.02"}
```
I'd remove the bound and check

Reply:
I see apologies. I think that may be the only one